src/App.tsx

```diff
@@ -1,25 +1,48 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import Pagination from './components/Pagination/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [perPage, setPerPages] = useState<number>(5);
```

Comment:
The name of the state setter function typically matches the state variable name, but with the 'set' prefix. Consider renaming 'setPerPages' to 'setPerPage' for consistency and to avoid confusion.